account/src/main/java/com/caldevsupplychain/account/util/UserMapper.java

```diff
@@ -1,16 +1,28 @@
 package com.caldevsupplychain.account.util;
 
+import org.mapstruct.IterableMapping;
 import org.mapstruct.Mapper;
 
 import com.caldevsupplychain.account.model.User;
 import com.caldevsupplychain.account.vo.UserBean;
 import com.caldevsupplychain.common.ws.account.UserWS;
 
+import java.util.List;
+
 @Mapper(config = MapperBaseConfig.class, uses = {RoleMapper.class})
 public interface UserMapper {
 
 	UserBean userWSToBean(UserWS userWS);
```

Comment:
You can simplify method names to 
```
UserBean toBean(UserWS userWS);
UserWS toWS(UserBean userBean);
...
List<UserBean> toBeans(List<UserWS> users);
List<UserWS> toWSs(List<UserBean> userBean);

```
As class name implies you are mapping `userXXX`. and param, return type tells you are converting from `<param type>` to `<return type>`

Reply:
There's some conflict when I do toBeans, mapstruct can't allow conflict name in this case which is weird, the iterable cannot have the same name, but the normal field can. For example: 

1. UserBean toBean(UserWS userWS);
2. UserBean toBean(User user);

These kind of field is accepted. However, I need to explicitly @Named them so iterable mapping on list can recognize them.